packs/static-site/config.yml

```diff
@@ -0,0 +1,12 @@
+name: Static Site Pack
+
+description: Helpful configuration for anyone running a static website.
```

Comment:
Can we add a bit more detail to the description, something like this?
`This observability packs helps to monitor a static website. It uses the New Relic Browser agent to measure frontend performance. It also use New Relic Synthetics to measure uptime of the static site.`

Also some high-level instructions:
`In order to use this pack, make sure to update the Synthetics script URL variable to point to your website`